_pages/channels/meme-battles.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: channel
+title: Meme-Battles
+channel: meme-battles
+permalink: /channels/meme-battles/
+parent: channels
+---
+
+A recreational channel to share and discuss your favorite memes, or even create new ones. This is a no-judgement zone. No meme is too old, no pepe too sad. 
+
+Drop in to take your mind off of things and have some good laughs and afterwards you get back to your code fully revitalized.
+
+![there should be a sweet meme here but it is broken](/images/longcat-tacgnol.png "It's an older meme but it checks out")
```

Comment:
Please use the alt text to describe the image. The title should have a period at the end.